test/all-tests.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
```

Comment:
This is almost same with `test/e2e-tests.sh`. Can we re-use it instead of creating this script?

Reply:
Can you suggest how to reuse it? I'm not able to find and elegant way. One way is to pass flag and change behaviour based on flags. But I'd rather have the behaviour clear from looking at the script.

Reply:
Sure. How about using env variable like this?

Makefile:

```
test-e2e:
        TEST_ALL="false" ./test/e2e-tests.sh

test-all:
        TEST_ALL="true" ./test/e2e-tests.sh
```

test/e2e-tests.sh
```
if [[ $TEST_ALL == "true" ]]; then
  (( !failed )) && install_serverless_previous || failed=2
  (( !failed )) && run_knative_serving_rolling_upgrade_tests || failed=3
  (( !failed )) && teardown_serverless || failed=4
fi
```

Reply:
Done. Thanks for suggestion.